### DISCUSSION TITLE
Log shows CRITICAL ( Certificate Hash not found )

### DISCUSSION THREAD

**Discussion by damionx7** [DNSCrypt/dnscrypt-proxy#1917] [Potential issues]
I just checked logs and it shows this CRITICAL warning. What to do ?
`dnscrypt-proxy[1382028]: [2021-10-05 20:50:43] [CRITICAL] [opennic-R4SAS-doh] Certificate hash [2954e7b866342b5221aaed4f6daf2be6f38444af8b7df35326ed6a972690bf78] not found`

**Comment by jedisct1** [comment]
This hash doesn't exist in current lists. Your `public-resolvers.md` file may be very outdated.

**Comment by damionx7** [comment]
Thanks @jedisct1 . I now updated the file.
